.github/workflows/ci.yml

```
@@ -0,0 +1,39 @@
+name: continuous-integration
+
+on:
+  [push]
```

(Same comment as at aiidalab-home) Is there a reason why we're not testing also on PRs?
```suggestion
on: [push, pull_request]
```

---

No.